### PR TITLE
TWM-156 implement homepage design

### PR DESCRIPTION
**Pull request by cdoyle-temple**
updates from meeting: 3/5/21

### CODE REVIEW

File: app/views/webpages/index.html.erb [hunk 1]
```diff
@@ -2,61 +2,70 @@
...
+      <%= link_to "#" do %>
+      <%= image_pack_tag "Illegal_Immigrants_Model_Minorities.jpg" %>
+        <p><strong>Illegal Immigrants Model Minorities</strong></p>
+      <% end %>
       </div>
       <div class="col-4 text-center">
-      <%= image_pack_tag "Women_s_Empowerment.jpg", style: "height:300px;max-width:200px;" %>
-        <p style="margin-top: 7px;color:#A50030"><strong>Women's Empowerment and Disempowerment in Brazil</strong></p>
+      <%= link_to "#" do %>
+      <%= image_pack_tag "Women_s_Empowerment.jpg" %>
```
Comment (sensei100) [inline app/views/webpages/index.html.erb:39]: @cdoyle-temple Do these images need alt tags? Or are you adding that later?

File: app/views/webpages/index.html.erb [hunk 2]
```diff
@@ -70,34 +79,36 @@
...
-              <a href="http://bit.ly/35mCxSg" target="_blank" rel="noopener">FLOP</a>&nbsp;&amp; Feb 8 at 
-              <a href="http://bit.ly/35PZqh7" target="_blank" rel="noopener">Big Blue Marble Bookstore</a>
-              </p>
+      <div class="media-body">
+        <h2>
+          <a href="/book/20000000010359">Do Right by Me</a>
+          <br><small><em>Learning to Raise Black Children in White Spaces</em></small>
+        </h2>
+        <p class="author">Valerie I. Harrison and Kathryn Peach D'Angelo</p>
+        <p style="line-height:1.2">
```
Comment (sensei100) [inline app/views/webpages/index.html.erb:94]: @cdoyle-temple I'm surprised to see in-line styles.  Is this temporary?

**Review by sensei100 — APPROVED** [review]

File: app/views/webpages/index.html.erb (hunk 1 above)
Comment (cdoyle-temple) [reply app/views/webpages/index.html.erb:39]: These images are treated as decoration as they supply no more information than the adjacent title text on the page. Screen readers skip such images with null alt values.

File: app/views/webpages/index.html.erb (hunk 2 above)
Comment (cdoyle-temple) [reply app/views/webpages/index.html.erb:94]: Missed this...

**Pull request merged**